training/general/src/environments/sklearn-1.1/tests/src/main.py

```diff
@@ -13,17 +12,12 @@
 
 # define functions
 def main(args):
-    """Run and evaluate model."""
-    # enable auto logging
-    mlflow.autolog()
 
     # setup parameters
     params = {
         "fit_intercept": args.fit_intercept,
-        "normalize": args.normalize,
```

Comment:
Why was the normalize param removed?

Reply:
Per latest docs of sklearn, https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LinearRegression.html, `normalize` is no more a parameter.